test/services/UpdateSave4LaterServiceSpec.scala

```diff
@@ -0,0 +1,264 @@
+      Some(amendVariationRenewalRespone))
+
+  }
+
+  "UpdateSave4LaterService" when {
+
+    "update is called" must {
+      "retrieve the specified file from stubs and update save for later with the contents" in new Fixture {
+
+        mockCacheSave[ViewResponse]
+        mockCacheSave[BusinessMatching]
+        mockCacheSave[Seq[TradingPremises]]
+        mockCacheSave[Seq[BankDetails]]
+        mockCacheSave[AddPerson]
+        mockCacheSave[BusinessActivities]
+        mockCacheSave[Tcsp]
+        mockCacheSave[Seq[ResponsiblePerson]]
+        mockCacheSave[Asp]
+        mockCacheSave[MoneyServiceBusiness]
+        mockCacheSave[Hvd]
+        mockCacheSave[Supervision]
+        mockCacheSave[AboutTheBusiness]
+        mockCacheSave[EstateAgentBusiness]
+        mockCacheSave[SubscriptionResponse]
+        mockCacheSave[AmendVariationRenewalResponse]
+
+
+        await(updateSave4LaterService.update(updateSave4LaterResponse))
+
+        Mockito.verify(mockCacheConnector).save[ViewResponse](eqTo(ViewResponse.key), any())(any(), any(), any())
+        Mockito.verify(mockCacheConnector).save[BusinessMatching](eqTo(BusinessMatching.key), any())(any(), any(), any())
+        Mockito.verify(mockCacheConnector).save[Seq[TradingPremises]](eqTo(TradingPremises.key), any())(any(), any(), any())
+        Mockito.verify(mockCacheConnector).save[Seq[BankDetails]](eqTo(BankDetails.key), any())(any(), any(), any())
+        Mockito.verify(mockCacheConnector).save[AddPerson](eqTo(AddPerson.key), any())(any(), any(), any())
+        Mockito.verify(mockCacheConnector).save[BusinessActivities](eqTo(BusinessActivities.key), any())(any(), any(), any())
+        Mockito.verify(mockCacheConnector).save[Tcsp](eqTo(Tcsp.key), any())(any(), any(), any())
+        Mockito.verify(mockCacheConnector).save[Seq[ResponsiblePerson]](eqTo(ResponsiblePerson.key), any())(any(), any(), any())
+        Mockito.verify(mockCacheConnector).save[Asp](eqTo(Asp.key), any())(any(), any(), any())
+        Mockito.verify(mockCacheConnector).save[MoneyServiceBusiness](eqTo(MoneyServiceBusiness.key), any())(any(), any(), any())
+        Mockito.verify(mockCacheConnector).save[Hvd](eqTo(Hvd.key), any())(any(), any(), any())
+        Mockito.verify(mockCacheConnector).save[Supervision](eqTo(Supervision.key), any())(any(), any(), any())
+        Mockito.verify(mockCacheConnector).save[AboutTheBusiness](eqTo(AboutTheBusiness.key), any())(any(), any(), any())
+        Mockito.verify(mockCacheConnector).save[EstateAgentBusiness](eqTo(EstateAgentBusiness.key), any())(any(), any(), any())
+        Mockito.verify(mockCacheConnector).save[SubscriptionResponse](eqTo(SubscriptionResponse.key), any())(any(), any(), any())
+        Mockito.verify(mockCacheConnector).save[AmendVariationRenewalResponse](eqTo(AmendVariationRenewalResponse.key), any())(any(), any(), any())
+      }
+    }
+
+    "getDataFromStubs is called" must {
+      "return the specified file contents from stubs" in new Fixture {
```

Comment:
There is a missing test here. If the test is no longer needed, it should be removed from the code.